GitUI/UserControls/RevisionGrid/Columns/RevisionGraphColumnProvider.cs

```diff
@@ -225,7 +225,12 @@ public override void OnVisibleRowsChanged(in VisibleRowRange range)
 
             // Keep an extra page in the cache
             _graphCache.AdjustCapacity((range.Count * 2) + 1);
-            Column.Width = CalculateGraphColumnWidth(range);
+            int width = CalculateGraphColumnWidth(range);
+            if (Column.Width != width)
+            {
+                Column.Width = width;
+                Column.DataGridView.InvalidateColumn(Column.Index);
```

Comment:
I would expect that chagning the width would invalidate, but should not be harmful.

Reply:
> I would expect that changing the width would invalidate

Me too, but it does not.

---

> There are some artifacts.

If you mean missing parts of secondary lanes (when scrolling up), I know that - but not yet how to resolve it.
All other artifacts should be fixed by #11480.